lib/docs/scrapers/ember.rb

```diff
@@ -20,13 +22,7 @@ class Ember < UrlScraper
 
     options[:trailing_slash] = false
 
-    options[:container] = ->(filter) do
-      if filter.base_url.path.start_with?('/api')
-        'main article'
-      else
-        'main'
-      end
-    end
+    options[:container] = 'main'
```

Comment:
Unwrap article for api (for reduced content, and to make various CSS selectors effective, such as `._page > h1:first-child`)

```suggestion
    options[:container] = ->(filter) do
      if filter.base_url.host.start_with?('api')
        'main article'
      else
        'main'
      endhost.start_with?('api')
    end
```

Reply:
The issue with unwrapping the content, is that not all pages in the Ember guides have an article. I originally wanted to have this be 'main article', but as it turned out not all pages have an article tag. There isn't much of a pattern, but I'll see if I can tune it a bit.

Reply:
I wound up reversing the original container rule as it works with the new docs better.